### DISCUSSION TITLE
About exiting mitmproxy shortcut keys.

### DISCUSSION THREAD

**Discussion by xzhacker123** [mitmproxy/mitmproxy#5736] [Ideas]
When using mitmproxy, I often accidentally press the capital 'Q' and exit directly without any prompt. Is there any way to improve it? 
like: Quit (yes, no)? Thanks~>_<~!

**Comment by mhils** [comment]
`Q` intentionally does not prompt, only `q` does. A simple fix would be to bind `Q` to another key instead. :)
